import_export/admin.py

```diff
@@ -249,10 +249,12 @@ def add_success_message(self, result, request):
         opts = self.model._meta
 
         success_message = _(
-            "Import finished, with {} new and " "{} updated {}."
+            "Import complete: {} new, {} updated, {} deleted and {} skipped {}"
```

Comment:
Please can we change 'complete' to 'finished' (sorry I changed my mind).  I think it will be easier for the translations.

Reply:
Sure, If I have any questions I'll contact you. I'll get on this now.